isofit/utils/apply_oe.py

```diff
@@ -21,15 +20,6 @@
 from isofit.core.common import envi_header, ray_start, ray_terminate
 from isofit.utils import analytical_line, empirical_line, extractions, segment
 
-warn(
```

Comment:
Warning on import means that _any_ import of this file triggers a warning, including just executing `$ isofit --help`. I moved the warning to `apply_oe()` so that the use of this function triggers the warning. Use of the feature is ultimately the behavior that should be warned about. In theory a user could be relying on some of the global variables, but they are pretty meaningless.